src/views/Opportunities/OpportunityComponents/OpportunityForm.jsx

```diff
@@ -223,6 +224,19 @@ const OpportunityForm = (props) => {
     });
   };
 
+    if (props.id) {
+      const searchedemployed = employerOptions.find((option) => option.label === props.employer.name);
```

Comment:
shouldn't this code run when there is are values in `employerOptions` array? Because this array is being initialized as an `[]` empty set and there would no values in side it.

I think this code should be run after we get the API response for employer options.